docs/current/sailbot_workspace/usage/setup.md

```diff
@@ -282,7 +283,32 @@ A couple things to note:
         [stopping a codespace](https://docs.github.com/en/codespaces/developing-in-codespaces/stopping-and-starting-a-codespace#stopping-a-codespace){target=_blank}
         - Delete codespaces that you do not plan to use anymore:
         [deleting a codespace](https://docs.github.com/en/codespaces/developing-in-codespaces/deleting-a-codespace#deleting-a-codespace){target=_blank}
-2. Follow the local setup instructions starting from [5. Open the workspace file](#5-open-the-workspace-file)
+2. click **Code → Codespaces → +**, a new tab will open, wait as the browser IDE sets up.
+
+3. Open `sailbot.code-workspace` and click **Open Workspace**,
+wait a moment as the work space sets up, the IDE should have a blue border.
+
+4. press `ctrl + shift + p` and type **Run Build Tasks** then select **Build All**.
```

Comment:
Just to be hardware agnostic, can you please change this to something like "Open the command palette (`ctrl + shift + p` on windows) ..."